internal/config/config.go

```diff
@@ -424,21 +424,26 @@ func parseAddressPool(p addressPool, bgpCommunities map[string]uint32) (*Pool, e
 	if len(p.Addresses) == 0 {
 		return nil, errors.New("pool has no prefixes defined")
 	}
+
```

Comment:
s/won/own/ in the commit msg

Reply:
Also lenght -> length

Reply:
Fixed (it's one of those things I never get right)